src/js/View/HeaderView.js

```diff
@@ -62,6 +63,9 @@ App.View.HeaderView = Backbone.View.extend({
       case 'cedus':
         logo = '../verticals/theme-cedus/img/cedus_logo_cab.png';
         break;
+      case 'aquagis':
```

Comment:
We must take advantage of customizing the header template and put the logo image inside of it.

Reply:
@neokore done 48477b5d868aa1751bb856bff03a37eb8581b9b8